src/main/java/jenkins/advancedqueue/PriorityConfiguration.java

```diff
@@ -110,6 +112,10 @@ public int compare(JobGroup.PriorityStrategyHolder o1, JobGroup.PriorityStrategy
 		} else {
 			priorityConfigurationMatrixHelper = new PriorityConfigurationMatrixHelper();
 		}
+
+		if (PriorityConfigurationPlaceholderTaskHelper.isPlaceholderTaskUsed()) {
```

Comment:
`PriorityConfigurationPlaceholderTaskHelper.isPlaceholderTaskUsed()`  return  false if `Jenkins.getInstance().getPlugin("workflow-durable-task-step") == null` it is actually a plugin existence check.